src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/LeggTilBrevmottakerModal.tsx

```diff
@@ -108,7 +105,7 @@ export const LeggTilBrevmottakerModal: React.FC<Props> = ({
                         </div>
                     </>
                 )}
-            </Modal.Content>
+            </Modal.Body>
```

Comment:
På denne komponenten har jeg ikke dratt ut knappene som andre steder fordi det var utrolig mye logikk for hvilke knapper som vises, så jeg syns det var enklere å bare la det forbli som før 😬